### ISSUE TITLE
Add to conda

### ISSUE THREAD

**Issue by tand826** [tand826/wsiprocess#1]
now wsiprocess is only on the pypi server. Might better be on the conda server

**Comment by tand826** [comment]
[Official tutorial](https://conda.io/projects/conda-build/en/latest/user-guide/tutorials/build-pkgs-skeleton.html)

## copy metadata from pypi

```
conda skeleton pypi wsiprocess
```

- skeleton gets requirements from setup.py, so I need to list the requirements in the install_requirement in setup.py. Without this, I needed to directly add requirements to the meta.yaml. 
- NOTICE :: libvips can be installed directly from conda, so anyway I need to configure meta.yaml.
- TODO :: README.md could not be properly converted in the conda

## build wsiprocess for anaconda

```
conda-build wsiprocess
```

- output directory was $HOME/.anyenv/envs/pyenv/versions/anaconda3-5.0.0/conda-bld/osx-64

## register wsiprocess to anaconda

```
anaconda upload $HOME/.anyenv/envs/pyenv/versions/anaconda3-5.0.0/conda-bld/osx-64/xxx.tar.gz
```

## all in shell scripts and for other python versions and other platforms

```shell
anaconda login
conda skeleton pypi wsiprocess --output-dir ./conda_tmp
for ver in 6 7 8
do
    # in the root directory of wsiprocess
    conda-build --python 3.${ver} . --output-folder ./conda_tmp
    conda convert --platform all ./conda_tmp/osx-64/wsiprocess-0.0.1-py3${ver}_0.tar.bz2 -o conda_tmp
    for platform in linux-32 linux-64 linux-aarch64 linux-armv6l linux-armv7l linux-ppc64le osx-64 win-32 win-64
    do
        anaconda upload ./conda_tmp/${platform}/wsiprocess-0.0.1-py3${ver}_0.tar.bz2
    done
done
rm -rf ./conda_tmp
```

## other memos

```
# variants option; works same as above, but requirements conflict and could not complete for 3.5
conda-build --variants "{python: [3.5, 3.6, 3.7]}"
```

**Comment by tand826** [comment]
Still not available for python 3.5 or lower. Check again later.
